[nmstate-1.3] ovsdb: Raise exception if no ovsdb plugin when required

cathay4t opened the pull request:
Raise `NmstateDependencyError` when no plugin is providing
`NmstatePlugin.PLUGIN_CAPABILITY_OVSDB_GLOBAL` but with ovsdb global
config desired.

Integration test case included and marked as tier1 as OpenShift require
this use case.

maiqueb reviewed (approved):
Thank you.

Nice improvement on the error msg.

cathay4t committed:
ovsdb: Raise exception if no ovsdb plugin when required

Raise `NmstateDependencyError` when no plugin is providing
`NmstatePlugin.PLUGIN_CAPABILITY_OVSDB_GLOBAL` but with ovsdb global
config desired.

Integration test case included and marked as tier1 as OpenShift require
this use case.

Signed-off-by: Gris Ge <fge@redhat.com>

cathay4t committed:
test: Install python3-nispor in ovs2.11 test

Signed-off-by: Gris Ge <fge@redhat.com>
